rts/Sim/MoveTypes/GroundMoveType.cpp

```diff
@@ -1761,8 +1761,8 @@ void CGroundMoveType::HandleObjectCollisions()
 		// NOTE:
 		//   use the collider's MoveDef footprint as radius since it is
 		//   always mobile (its UnitDef footprint size may be different)
-		const float colliderFootPrintRadius = colliderMD->CalcFootPrintMaxInteriorRadius(); // ~= CalcFootPrintMinExteriorRadius(0.75f)
-		const float colliderAxisStretchFact = colliderMD->CalcFootPrintAxisStretchFactor();
+		const float colliderFootPrintRadius = collider->CalcFootPrintMaxInteriorRadius(); // ~= CalcFootPrintMinExteriorRadius(0.75f)
```

Comment:
I would say this is not a good business at all... Look the comment...

The path finder never will use the unit footprint. If the unit footprint is smaller than the corresponding movedef one, then it would be perfectly feasible to get stucked units. i.e. They are parking close together, and then pathfinding is classifying them as blocked, not letting them to move anymore